packages/flutter_tools/templates/app/lib/main.dart.tmpl

```diff
@@ -49,14 +49,21 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         // This is the theme of your application.
         //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
+        // TRY THIS: Try running your application with "flutter run". You'll see
+        // the application has a blue toolbar. Then, without quitting the app,
+        // try changing the seedColor in the colorScheme below to Colors.green
+        // and then invoke "hot reload" (save your changes or press the "hot
+        // reload" button in a Flutter-supported IDE, or press "r" if you used
+        // the command line to start the app).
+        //
         // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+        // state is not lost during the reload. To reset the state, use hot
+        // restart instead.
```

Comment:
for a new user who doesn't know what "application state", "reload", or "hot restart" are, I think the original "the application is not restarted" is probably clearer? This is the kind of change I'd love to test though.

Reply:
I changed it because it seemed to me that "restarted" and "reloaded" are not really all that distinct for a new user, and that this wording would help to define the terms for them.

If the new user didn't understand yet that there was a distinction between reload and restart, thinking they both meant something like "start again", then the original could be read as "hit the start again button ... and the app is not started again", which would be confusing.

I agree that it does introduce the "application state" term. I couldn't think of another way to say that, though.

So would you rather just leave the original until we've done testing here then?

Reply:
i don't feel strongly one way or the other (other than being just inherently defensive and suspicious of any changes to this file). i agree that in the absence of data, either might be better.